extensions/amp-lightbox-viewer/0.1/service/lightbox-manager-impl.js

```diff
@@ -83,23 +102,39 @@ export class LightboxManager {
    */
   scanLightboxables_() {
     return this.ampdoc_.whenReady().then(() => {
+      const viewer = elementByTag(this.ampdoc_.getRootNode(), VIEWER_TAG).id;
       const matches = this.ampdoc_.getRootNode().querySelectorAll('[lightbox]');
-      this.elements_ = [];
       for (let i = 0; i < matches.length; i++) {
         const element = matches[i];
-        if (element.getAttribute('lightbox').toLowerCase() != 'none') {
-          this.elements_.push(element);
+        if (element.tagName == 'AMP-CAROUSEL') {
+          // TODO: special case carousel
+        } else {
+          const lightboxGroupId = element.getAttribute('lightbox');
+          if (lightboxGroupId == '' || lightboxGroupId == 'default') {
```

Comment:
The nested `if` statements here make it difficult to read quickly.

Consider simplifying to something similar to this:

```js
const groupId = element.getAttribute('lightbox') || 'default';
if (!this.lightboxGroups_[groupId]) {
  this.lightboxGroups_[groupId] = [];
}
this.lightboxGroups_[groupId].push(element);
```